tests/integration/netbox-deploy.py

```diff
@@ -33,7 +34,6 @@ def make_netbox_calls(endpoint, payload):
         created = endpoint.create(payload)
     except pynetbox.RequestError as e:
         print(e.error)
-        ERRORS = True
```

Comment:
How come this was removed? No discussion in the commit message and this appears to alter behavior

Reply:
Basically ERRORS now is always `False` - which means the logic the bottom that checks for `ERRORS` never executes even if there is an error.

Reply:
I swore I did not delete it even though sourcery suggested it, thanks for the catch.